airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java

```diff
@@ -196,11 +233,11 @@ private static void runConsumer(final AirbyteMessageConsumer consumer) throws Ex
    */
   @VisibleForTesting
   static void watchForOrphanThreads(final Procedure runMethod,
-                                    final Runnable exitHook,
```

Comment:
these tab/space amounts probably shouldn't be changed to match how other method's space out their arguments. Another benefit in this case is that it's easier to see that this method throws an Exception

Reply:
Thanks, I ran the formatting command on these files, so I'll defer to that